lib/src/Base/Stat/GeneralizedExponential.cxx

```diff
@@ -78,6 +78,7 @@ Scalar GeneralizedExponential::computeAsScalar(const Point & tau) const
     tauOverThetaNorm += dx * dx;
   }
   tauOverThetaNorm = sqrt(tauOverThetaNorm);
+  const CovarianceMatrix outputCovariance(outputCovariance_);
   return tauOverThetaNorm <= SpecFunc::ScalarEpsilon ? outputCovariance_(0, 0) * (1.0 + nuggetFactor_) : outputCovariance_(0, 0) * exp(-pow(tauOverThetaNorm, p_));
```

Comment:
indeed!
this commit avoids a crash in the KLQuadrature example, and also unnecessary copyonwrite, because the compiler chose the non-const accessor (this attribute is mutable)